length_field_based_frameconn.go

```diff
@@ -147,6 +169,9 @@ func (fc *lengthFieldBasedFrameConn) WriteFrame(p []byte) error {
 		if length >= 256 {
 			return fmt.Errorf("length does not fit into a byte: %d", length)
 		}
+		if fc.encoderConfig.Format == ASCII {
```

Comment:
It really does not make sense to have 1 byte ascii, as it will only contain less than 10 bytes, so I disallowed it.